.pinned

```diff
@@ -1,15 +1,17 @@
 bearssl;https://github.com/status-im/nim-bearssl@#a647994910904b0103a05db3a5ec1ecfc4d91a88
 chronicles;https://github.com/status-im/nim-chronicles@#32ac8679680ea699f7dbc046e8e0131cac97d41a
-chronos;https://github.com/status-im/nim-chronos@#75d030ff71264513fb9701c75a326cd36fcb4692
+chronos;https://github.com/status-im/nim-chronos@#e9f8baa6ee2e21ff8e6b6c0ce0c22368cdd9e758
+combparser;https://github.com/PMunch/combparser@#ba4464c005d7617c008e2ed2ebc1ba52feb469c6
```

Comment:
not needed

Reply:
@arnetheduck I am reopening this comment as it appears that combparser is indeed needed.
It's used by protobuf-ser [here](https://github.com/status-im/nim-protobuf-serialization/blob/master/protobuf_serialization/files/proto_parser.nim) and is therefore installed / needed by the nim-libp2p. I don't know if it's an oversight on your part or if you eventually want to remove the use of combparser.
Or maybe I'm just missing something.

Reply:
as long as you don't use `import_proto3` it's not needed - ie that file should not be imported by default, per the nps readme.

Reply:
The pinned file is generated from the nimble dependency, so it will still be bringed in, as long as nimble doesn't support optional deps